dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/transport/http/URLConnectionHTTPConduit.java

```diff
@@ -463,6 +470,30 @@ protected boolean usingProxy() {
         protected void setFixedLengthStreamingMode(int i) {
             // [CXF-6227] do not call connection.setFixedLengthStreamingMode(i)
             // to prevent https://bugs.openjdk.java.net/browse/JDK-8044726
+            
+            //Allow this mode to be set only if the Expect header has been set to "100-continue"           
+            if ((message != null) && (allowFixedLengthStreamMode == null)) {
```

Comment:
if you use `outMessage` instead of `message` then you will get the Message instance associated with this output stream.